sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateService.java

```diff
@@ -111,8 +111,7 @@
                                                        Context context);
 
     @Get("certificates/{certificate-name}/{certificate-version}")
-    @ExpectedResponses({200, 404})
-    @UnexpectedResponseExceptionType(code = {403}, value = ResourceModifiedException.class)
+    @ExpectedResponses({200, 404, 403})
```

Comment:
we need to do this in beginCreateCertificate Polling operation service call, not here.